client/dist/src/components/bars.jsx

```diff
@@ -1,37 +1,93 @@
 import React from 'react';
 import axios from 'axios';
+import Styles from '../css/bars.css';
+
+
+const calculateAverage = (response) => {
+  var total = 0;
+  for (let i = 0; i < response.data.length; i++) {
+    total += response.data[i].prices;
+  }
+  return total / response.data.length;
+}
+
+const findCurrentPrice = (response) => {
+  var currentPrice = 0;
+  for (let i = 0; i < response.data.length; i++) {
+    if (response.data[i].dates === 29) {
+      currentPrice = response.data[i].prices;
+    }
+  }
+  return currentPrice;
+}
+
+const marketIsOpen = () => {
+  var date = new Date();
+  var currentHour = date.getHours();
+  if (currentHour > 15 && currentHour < 6) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 class Bars extends React.Component {
     constructor(props) {
       super(props)
       this.state = {
         data: [],
+        currentPrice: 0,
+        average: 0,
+        marketHours: this.props.marketIsOpen,
       }
     }
     componentDidMount(props) {
-      axios.get('/company/6')
+      axios.get('/company/10')
         .then((response) => {
-          console.log(response.data);
-          console.log(response.data[0].volume);
+          let average = calculateAverage(response);
+          let currentPrice = findCurrentPrice(response);
+          // let isMarketOpen = marketIsOpen();
           this.setState({
-            data: response.data
+            data: response.data,
+            currentPrice,
+            average
           });
         })
         .catch((error) => {
           console.log(error);
         })
     }
-    render() {
-      return (
-        <div className="barContainer">
-      {this.state.data.map((el)=> {
-        return <div className='bar'style={{height:`${el.volume}%`}} key={el._id}></div>
-      })
-  } <
-  /div>
-)
 
-}
+
+    render() {
+        if (this.state.currentPrice > this.state.average) {
+          var maxHighlightBar = this.state.currentPrice;
```

Comment:
a lot of use of var here, if it is not needed outside of this block scope change it over to let or const. I might also suggest installing eslint because it will make sure to remind you of this.